stories/topics/proc-aap-gcp-application-upgrade.adoc

```diff
@@ -0,0 +1,88 @@
+[id="proc-aap-gcp-application-upgrade"]
+
+Technical Preview
+~~~~~~~~~~~~~~~~~
+
+You can upgrade your existing {AAPonGCP} deployment to the newer version. The upgrade process covers upgrading of automation hub and automation controller. The upgrade process takes roughly the same amount of time as a {AAPonGCP} deployment install. Follow the steps below to ensure a smooth upgrade process.
+
+[NOTE]
+=====
+{AAPonGCP} supports sequential upgrades. All upgrades should be no more than one major version behind what you are currently upgrading to. For example, in order to upgrade to {AAPonGCP} to 2.2.20230215, you must be on the first version of {AAPonGCP}.
+=====
+
+.Prerequisites
+* You require a docker installed in order to run the upgrade playbook.
```

Comment:
@leena-jawale: Is there a minimum version supported or to be specified here?

could be revised as:

```
- docker with minimum version x.y to run the playbooks
```

Reply:
I tested using:
- Fedora 37
- Docker version 23.0.1, build a5ee5b1